pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -156,7 +145,7 @@ func CheckSafeToScaleCluster(
 		return err
 	}
 
-	klog.V(4).Infof("node count %d satisfies minimum of %d required by the %s bootstrap scaling strategy", nodeCount, minimumNodes, scalingStrategy)
+	klog.V(4).Infof("node count satisfies minimum of %d required by the %s bootstrap scaling strategy", minimumNodes, scalingStrategy)
```

Comment:
>  len(memberHealth.GetHealthyMembers())

could also do as a replacement for node count?

Reply:
done